charts/netbird/values.yaml

```diff
@@ -163,7 +163,7 @@ management:
 
     ## @param management.image.pullPolicy
     ##
-    pullPolicy: IfNotPresent
+    pullPolicy: Always
```

Comment:
This should be added as a helm parameter so user can decide what best fits in their usecase

Reply:
reverting